### ISSUE TITLE
hide/filter out files with X pattern

### ISSUE THREAD

**Issue by ghost** [ranger/ranger#1313]
Is it possible to filter out files called X? Say for example you want to hide normally hidden files from a windows partition such as desktop.ini, can you filter those out using regex or something?

**Comment by Vifon** [comment]
Yes, you can do this with the new filter stack in ranger 1.9.2. First add a new filter matching all the files you want to hide with <kbd>.n</kbd> and then negate it with <kbd>.!</kbd>.
